packages/tasks/src/hardware.ts

```diff
@@ -176,6 +176,10 @@ export const SKUS = {
 				tflops: 28.26,
 				memory: [32, 16],
 			},
+			"GTX 1650 Mobile": {
+				tflops: 3.1,
```

Comment:
```suggestion
				tflops: 6.39,
```

We are using the `float16` numbers as a reference :)

Reply:
Hmm, OK :)